packages/instant/src/containers/selected_erc20_asset_amount_input.ts

```diff
@@ -79,11 +80,24 @@ const updateBuyQuoteAsync = async (
         newBuyQuote = await assetBuyer.getBuyQuoteAsync(asset.assetData, baseUnitValue);
     } catch (error) {
         dispatch(actions.setQuoteRequestStateFailure());
-        errorUtil.errorFlasher.flashNewError(dispatch, error);
+        let errorMessage;
+        if (error.message === AssetBuyerError.InsufficientAssetLiquidity) {
```

Comment:
I think ideally this would still be in a separate util method and we would keep the tests around